src/main/scala/com/campudus/tableaux/database/model/structure/ColumnModel.scala

```diff
@@ -930,23 +1058,24 @@ class ColumnModel(val connection: DatabaseConnection)(
     val identifier = row.get[Boolean](5)
     val separator = row.get[Boolean](6)
     val attributes = new JsonObject(row.get[String](7))
+    val rules = new JsonArray(row.get[String](8))
 
     val languageType = LanguageType(Option(row.get[String](4))) match {
       case LanguageNeutral => LanguageNeutral
       case MultiLanguage => MultiLanguage
       case c: MultiCountry =>
-        val codes = Option(row.get[String](8))
+        val codes = Option(row.get[String](9))
           .map(str => Json.fromArrayString(str).asScala.map({ case code: String => code }).toSeq)
           .getOrElse(Seq.empty[String])
 
         MultiCountry(CountryCodes(codes))
     }
 
-    val groupColumnIds = Option(row.get[String](9))
+    val groupColumnIds = Option(row.get[String](10))
       .map(str => Json.fromArrayString(str).asScala.map(_.asInstanceOf[Int].toLong).toSeq)
       .getOrElse(Seq.empty[ColumnId])
 
-    val formatPattern = Option(row.get[String](9))
+    val formatPattern = Option(row.get[String](10))
```

Comment:
**[major]** - Mich wundert, dass bei dir die Tests lokal laufen. Bei mir tun sie das nicht. Vll. lässt du sie auch auf Basis irgendeines caches laufen. Direkt per gradle failen die bei mir. z.B.
```bash
./gradlew test --tests="com.campudus.tableaux.api.structure.GroupColumnTest.createTripleGroupColumnWithValidFormat"
```
Jedenfalls hat sich der Fehler schon in einem früheren feature eingeschlichen und wurde nur weiter verschleppt.
Da die columns `attributes` und `rules` früher im select stehen, verschiebt sich alles und `formatPattern` wurde einmal vergessen anzupassen. Wenn das von array pos 11 geholt wird, passt wieder alles und die Tests laufen

Reply:
Ich hab jetzt auch mal Zeit gefunden, die travis Integration auf github actions umzustellen. Das sollte mit dem [PR-241 ](https://github.com/campudus/tableaux/pull/241) dann auch wieder alles funktionieren. Vll. können wir den noch vor deinem mergen und den flow ausprobieren.